library/src/main/java/com/devbrackets/android/exomedia/core/api/VideoViewApi.java

```diff
@@ -94,6 +94,8 @@
 
     void setScaleType(@NonNull ScaleType scaleType);
 
+    void setIgnoreAspectRatio(boolean ignoreAspectRatio);
```

Comment:
I think this is a little vague... for what reason would I "IgnoreAspectRation()" (coming from an external users point of view).  Maybe `setIgnoreAspectRatioInLayout()`?